source/m2minterfaceimpl.cpp

```diff
@@ -841,6 +873,7 @@ void M2MInterfaceImpl::state_unregister( EventData */*data*/)
     if(!_nsdl_interface.send_unregister_message()) {
         tr_error("M2MInterfaceImpl::state_unregister : M2MInterface::NotRegistered");
```

Comment:
could we use same error messages in traces as well - now same thing is declared twice. Safe also some memory ;)